Threat connect v2 bug (#19746)
demisto/content: Packs/ThreatConnect/ReleaseNotes/2_0_19.md

#### Integrations
##### ThreatConnect v2

> - Fixed an issue where the *ip* command did not work as intended.

Reviewer: ```suggestion
- Fixed an issue where the ***ip*** command failed.
```
